spyder/widgets/sourcecode/editorextensions/manager.py

```diff
@@ -16,17 +17,16 @@
 
 
 class EditorExtensionsManager(Manager):
-    """
-    Manages the list of editor extensions of the CodeEdit widget.
-    """
+    """Manages the list of editor extensions of the CodeEdit widget."""
 
     def __init__(self, editor):
+        """Initialize and add a reference to the editor."""
         super(EditorExtensionsManager, self).__init__(editor)
         self._extensions = {}
 
     def append(self, extension):
```

Comment:
I didn't see your response because It was collapsed, I'm going to change it for `add`